planet-express/java/src/Crew.java

```diff
@@ -0,0 +1,74 @@
+
+public class Crew {
+	int thirst = 50;
```

Comment:
Sorry, I mean for the `50`s. If you wanted to change the start value for all properties, you could just change the constant one time.